#!/usr/bin/env python
# human pose prediction configuration

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t

gen = ParameterGenerator()

gen.add("scale_lower", double_t, 0, "human slow-down velocity multiplier", 0.8, 0.0, 1.0)
gen.add("scale_nominal", double_t, 0, "human nominal velocity multiplier", 1.0, 0.0, 10.0)
gen.add("scale_higher", double_t, 0, "human speed-up velocity multiplier", 1.2, 1.0, 10.0)
gen.add("angle", double_t, 0, "deviation angle for human position predictions", 0.1, 0.0, 3.14)

exit(gen.generate("hanp_prediction", "human_pose_prediction", "HumanPosePrediction"))
